src/main/resources/db-scripts/clickhouse/clickhouse.sql

```diff
@@ -376,7 +414,7 @@ FROM
             JOIN genetic_profile_samples gps ON gp.genetic_profile_id = gps.genetic_profile_id
             JOIN gene g ON ga.genetic_entity_id = g.genetic_entity_id
         WHERE
-             gp.genetic_alteration_type NOT IN ('GENERIC_ASSAY', 'MUTATION_EXTENDED', 'STRUCTURAL_VARIANT'))
+             gp.genetic_alteration_type NOT IN ('GENERIC_ASSAY', 'MUTATION_EXTENDED', 'MUTATION_UNCALLED', 'STRUCTURAL_VARIANT'))
```

Comment:
@sheridancbio this is intentional?

Reply:
Yes. The profile type "MUTATION_UNCALLED" is present in the public portal database and was not account for previously. These events should not be attempted to be put into genetic_alteration_derived.